src/main/java/org/zone/commands/structure/region/flags/display/greetings/ZoneFlagGreetingsDisplaySetTitleCommand.java

```diff
@@ -35,12 +36,12 @@
                     ZonePermissions.OVERRIDE_FLAG_GREETINGS_MESSAGE_DISPLAY_SET_TITLE));
     public static final OptionalArgument<Component> SUB_TITLE =
             new OptionalArgument<>(new ComponentRemainingArgument("subTitle"), (Component) null);
-    public static final OptionalArgument<Integer> FADE_IN =
-            new OptionalArgument<>(new IntegerArgument("fadeIn"), (Integer) null);
-    public static final OptionalArgument<Integer> STAY =
-            new OptionalArgument<>(new IntegerArgument("stay"), (Integer) null);
-    public static final OptionalArgument<Integer> FADE_OUT =
-            new OptionalArgument<>(new IntegerArgument("fadeOut"), (Integer) null);
+    public static final RangeArgument<Integer> FADE_IN = RangeArgument.createArgument("fadeIn", 0
+            , Integer.MAX_VALUE);
+    public static final RangeArgument<Integer> STAY = RangeArgument.createArgument("stay", 0,
+            Integer.MAX_VALUE);
+    public static final RangeArgument<Integer> FADE_OUT = RangeArgument.createArgument("fadeOut",
+            0, Integer.MAX_VALUE);
```

Comment:
Ill let it pass, but i still dont like how many forced arguments there are

Reply:
K

Reply:
Btw how would u do it instead?